backend/src/BuiltinExecution/Libs/Char.fs

```diff
@@ -54,8 +54,9 @@ let fns : List<BuiltInFn> =
       fn =
         function
         | _, _, [ DChar c ] ->
-          if c.Length = 1 then
-            Dval.optionSome (DInt(System.Char.ConvertToUtf32(c, 0))) |> Ply
+          let charValue = System.Char.ConvertToUtf32(c, 0)
```

Comment:
While this works, it feels like overkill. I asked chatgpt about how best to do this, and it suggested `(int)c`. That seems to me like it would work.

Conversation here: https://chat.openai.com/share/2a6f10f0-d45d-4e71-ac57-584570baeda8